src/display/annotation_layer.js

```diff
@@ -556,6 +565,99 @@ class LinkAnnotationElement extends AnnotationElement {
     }
     link.className = "internalLink";
   }
+
+  _bindResetFormAction(link, resetForm) {
+    if (!this._fieldObjects) {
+      warn(
+        `_bindResetFormAction - "resetForm" action not supported, ` +
+          "ensure that the `fieldObjects` parameter is provided."
+      );
+      return;
+    }
+
+    link.className = "internalLink";
+    const otherClickAction = link.onclick;
+    link.onclick = () => {
+      if (otherClickAction) {
+        otherClickAction();
+      }
+
+      const {
+        fields: resetFormFields,
+        refs: resetFormRefs,
+        include,
+      } = resetForm;
+
+      const allFields = [];
+      if (resetFormFields.length !== 0 || resetFormRefs.length !== 0) {
+        const fieldIds = new Set(resetFormRefs);
+        for (const fieldName of resetFormFields) {
+          const fields = this._fieldObjects[fieldName] || [];
+          for (const { id } of fields) {
+            fieldIds.add(id);
+          }
+        }
+        for (const fields of Object.values(this._fieldObjects)) {
+          for (const field of fields) {
+            if (include === fieldIds.has(field.id)) {
```

Comment:
Nit: Let's swap the order in this comparison, since the "constant" value should usually be on the right. (Note that a comparison such as `if (5 === someVar) { ... }` would actually be a linting failure.)